RCTAppleHealthKit/RCTAppleHealthKit+Methods_ReproductiveHealth.h

```diff
@@ -0,0 +1,10 @@
+#import "RCTAppleHealthKit.h"
+
+@interface RCTAppleHealthKit (Methods_Vitals)
+
+- (void)reproductiveHealth_getBasalBodyTemperatureSamples:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
+- (void)reproductiveHealth_getMenstruationFlowSamples:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
+- (void)reproductiveHealth_getOvulationTestResultSamples:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
+- (void)reproductiveHealth_getSexualActivitySamples:(NSDictionary *)input callback:(RCTResponseSenderBlock)callback;
+
```

Comment:
Why do you add this prefix here? They are already in a class `ReproductiveHealth` I find it somehow repetitive.

Reply:
Consistency with the other modules, like activity or vital.

Reply:
Aham, it seems I've been looking to different modules where this convention is not followed.